river/XdgToplevel.zig

```diff
@@ -349,15 +350,18 @@ fn handleRequestMove(
     const self = @fieldParentPtr(Self, "request_move", listener);
     const seat = @intToPtr(*Seat, event.seat.seat.data);
     if ((self.view.pending.float or self.view.output.pending.layout == null) and !self.view.pending.fullscreen)
-        seat.cursor.enterMode(.move, self.view);
+        seat.cursor.enterMode(.move, self.view, null);
 }
 
 /// Called when the client asks to be resized via the cursor.
 fn handleRequestResize(listener: *wl.Listener(*wlr.XdgToplevel.event.Resize), event: *wlr.XdgToplevel.event.Resize) void {
     const self = @fieldParentPtr(Self, "request_resize", listener);
     const seat = @intToPtr(*Seat, event.seat.seat.data);
     if ((self.view.pending.float or self.view.output.pending.layout == null) and !self.view.pending.fullscreen)
-        seat.cursor.enterMode(.resize, self.view);
+        seat.cursor.enterMode(.resize, self.view, Cursor.ResizeDirection.fromWlrEdges(event.edges) catch {
+            log.err("Client requested resize with invalid edge values. The request will be ignored.", .{});
```

Comment:
I feel like this should be a protocol error but I don't see any error enum value that would match this. I asked in `#wayland`.

Reply:
https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/123

Reply:
I'll add a todo comment there.